.github/workflows/types.yml

```diff
@@ -0,0 +1,34 @@
+name: Types
```

Comment:
Similar to https://github.com/mozilla/pdf.js/pull/17263/files, let's call the file `types_tests.yml` and this name `Types tests`. I'm using this naming scheme so that in the workflow rundown it's clear which workflows involve tests and which involve actions (such as linting or publishing the website), and I'm planning on making every kind of test (type tests, font tests, integration tests, unit tests) a separate workflow.